### PR TITLE
Little improvements in the README: update FAQ link and add Slack channel

### PR DESCRIPTION
**Pull request by matks**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Little improvements in the README
| Type?             | improvement
| Category?         | PM
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| How to test?      | 
| Possible impacts? | 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/23467)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ First-time users will be particularly interested in the following guides:
 * [User Guide][user-guide]: All there is to know to put PrestaShop to good use.
 * [Updating Guide][updating-guide]: Switching to the newest version is not trivial. Make sure you do it right.
 * [Merchant's Guide][merchant-guide]: Tips and tricks for first-time online sellers.
-* The [FAQ][faq-17] and the [Troubleshooting][troubleshooting] pages should also be of tremendous help to you.
+* The [Troubleshooting][troubleshooting] page should also be of tremendous help to you.
```
Comment (matks) [inline README.md]: This FAQ is 5 years old. It is now outdated.

File: README.md [hunk 2]
```diff
@@ -139,7 +140,6 @@ Thank you for downloading and using the PrestaShop Open Source e-commerce soluti
 [user-guide]: https://doc.prestashop.com/display/PS17/User+Guide
 [updating-guide]: https://doc.prestashop.com/display/PS16/Updating+PrestaShop
 [merchant-guide]: https://doc.prestashop.com/display/PS16/Merchant%27s+Guide
-[faq-17]: https://build.prestashop.com/news/prestashop-1-7-faq/
```
Comment (PierreRambaud) [inline README.md:142]: https://devdocs.prestashop.com/1.7/faq/ :)

File: README.md (hunk 1 above)
Comment (Progi1984) [reply README.md]: But we have a FAQ in devdocs ?

**Review by Progi1984 — COMMENTED** [review]
A feedback for the FAQ

**Comment by matks** [comment]
@Progi1984 @PierreRambaud Thanks, I did not think of the FAQ in devdocs.

PR updated, commit ammended. PR title improved.

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @matks

**Pull request merged**